pkg/security/probe/probe_unsupported.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+// +build linux
```

Comment:
There is an extra `// +build` comment here

Reply:
Yes.  that is an artifact of a bug in the `go generate` stage.  it either puts it there, or before the copyright, which in turn fails the copyright linting.

I have raised this issue with CWS for a future PR, but objective is to have the actual tests pass.

Reply:
Where is this file being generated from?

Reply:
`pkg/security/secl`

Reply:
I'm going to take a look at what is generating this

Reply:
This is NOT an artifact of a bug in the `go generate` stage, this file is not even generated

Reply:
sorry.  You're right.  this one I think is copy paste error.  @paulcacheux